### ISSUE TITLE
Custom image requires template edits to param sourceImage - resource cannot be found

### ISSUE THREAD

**Issue by JeffGiroux** [F5Networks/f5-google-gdm-templates#47]
## Do you already have an issue opened with F5 support?
No

## Description
I want to use a custom image I uploaded to Google into my project. The current template python code is coded to populate sourceImage with a path containing the F5 public project instead of my project. As a result, the GDM template fails and says image is not found.

Please provide a new parameter to allow custom user image IDs. If value is present, then python code should substitute project ID accordingly in sourceImage value below.

Code in question:
```
'sourceImage': ''.join([COMPUTE_URL_BASE, 'projects/f5-7626-networks-public',
  '/global/images/',
  context.properties['imageName'],
  ])
```

# Workaround = edit python code, replace project ID
```
'sourceImage': ''.join([COMPUTE_URL_BASE, 'projects/myproject123',
  '/global/images/',
  context.properties['imageName'],
  ])
```

## Template
All google templates

## Severity Level
3

**Comment by shyawnkarim** [comment]
Thanks for reporting this enhancement request. I've created Jira ticket, internal ID ESECLDTPLT-2031.

**Comment by shyawnkarim** [comment]
Closing due to age. These legacy templates are now in maintenance mode and are being replaced by our next-generation templates available in the [Cloud Templates 2.0 GitHub repo](https://github.com/F5Networks/f5-google-gdm-templates-v2).
